js/speaker.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(function() {
+  fetch('../data/speakers.json')
+    .then(response => response.json())
```

Comment:
Please do not use arrow functions. We are not compiling the javascript code. Just use `function` instead.

Reply:
@ananyo2012 It's the ES2016 standard, and it should be followed 😅. But if you want I will remove

Reply:
Yes if you are compiling the code then you should follow ES6. But we are not doing any compilation here. So you have to use the basic syntax as ES6 is still not supported in many browsers.